04/app.js

```diff
@@ -1,89 +1,99 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
+import React from "react";
+import { createRoot } from "react-dom/client";
 
-const root = createRoot(document.querySelector('#root'));
+const root = createRoot(document.querySelector("#root"));
 
 class App extends React.Component {
-    state = { 
-        firstName: '',
-        lastName: '',
-        searchQuery: '',
-        users: ['Jan Kowalski', 'Michał Nowak'],
-    }
+	state = {
+		firstName: "",
+		lastName: "",
+		searchQuery: "",
+		users: ["Jan Kowalski", "Michał Nowak"],
+	};
 
-    renderUsersList() {
-        const {users} = this.state;
-        return users.map(name => {
-            return (
-                <li onClick={ this.clickHandler }>
-                    { name }
-                </li>
-            );
-        });
-    }
+	renderUsersList() {
+		const { users, searchQuery } = this.state;
 
-    clickHandler = e => {
-        const {innerText: userName} = e.target;
-        this.removeUser(userName);
-    }
+		if (searchQuery !== "") {
+			const filteredUsers = users.filter(user => {
+				return user === searchQuery;
```

Comment:
Tutaj jest błąd - nie chodzi o dokładne odwzorowanie, a o to czy się "zawiera" jeden w drugim. Należałoby użyć `.includes()`